docker-compose-centos.yaml

```diff
@@ -0,0 +1,123 @@
+# Copyright 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '3.5'
+
+services:
+  avalon-shell:
+    container_name: avalon-shell
+    image: avalon-shell-dev
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile.centos
```

Comment:
Is "dockerfile" path the only difference between centOS and ubuntu. If so, we can optimize docker-compose files.

Reply:
Yes, this can be done by having an variable defined in .env file. Eg:
DISTRO=bionic or centos

in docker-compose file:
dockerfile: ./docker/Dockerfile.${DISTRO}